src/Features/Core/Portable/ReplaceMethodWithProperty/ReplaceMethodWithPropertyCodeRefactoringProvider.cs

```diff
@@ -95,14 +95,20 @@ private static bool HasGetPrefix(SyntaxToken identifier)
 
         private static bool HasGetPrefix(string text)
         {
-            return text.StartsWith(GetPrefix) && text.Length > GetPrefix.Length && !char.IsLower(text[GetPrefix.Length]);
+            return HasPrefix(text, GetPrefix);
+        }
+        private static bool HasPrefix(string text, string prefix)
+        {
+            return text.StartsWith(prefix, StringComparison.OrdinalIgnoreCase) && text.Length > prefix.Length && !char.IsLower(text[prefix.Length]);
```

Comment:
VB, at least, has special rules for historical compiler bugs, which is why the compiler API exists. I believe in general the answer for VB at least is one should never use OrdinalIgnoreCase because the compiler itself is not using that. We violate this everywhere.